app/welcome/first-app/route.js

```diff
@@ -1,23 +1,8 @@
 import Ember from 'ember';
-import { read, write } from '../../utils/storage';
+import { write } from '../../utils/storage';
 export var firstAppKey = '_aptible_firstAppData';
 
 export default Ember.Route.extend({
-  model: function(){
-    var firstApp = read(firstAppKey);
-
-    if (firstApp) {
-      return firstApp;
-    } else {
-      return {
-        appHandle: '',
-        dbHandle: '',
-        diskSize: 10,
-        dbType: null
-      };
-    }
-  },
```

Comment:
This code was responsible for restoring the app and db as entered if a user returned to the wizard. And in general it provides defaults for the initial screens.

It seems like it should not be dropped?